<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QMainWindow" name="SimulationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="planeView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>856</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="profileView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>600</y>
      <width>856</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>360</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> PLANE VISIBILITY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="geo_base_en">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>410</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>geo base</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="node_points_en">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>450</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>node points</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="text">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>570</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>text</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="lines">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>490</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>segments</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="nodeDetails">
    <property name="geometry">
     <rect>
      <x>1116</x>
      <y>0</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/icons/img/deleteImg.png) ;
background-position: center;
  
border: 2px outset #094065;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addTurnNodeButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>550</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/icons/img/dot.png) ;
background-position: center;
border: 2px outset #094065;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addNodeToSegmentButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>410</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/icons/img/line.png) ;
background-position: center;
border: 2px outset #094065;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectItemButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>480</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/icons/img/pointer.png) ;
background-position: center;
border: 2px outset #094065;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="moveNodeButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>620</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/new/icons/img/handIconjpg.png) ;
background-position: center;
border: 2px outset #094065;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>841</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>730</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(132, 132, 132);
border: 2px outset #094065;
border-radius: 15px;                     /* &lt;----  20px  */ 
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simulateButton">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>730</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(132, 132, 132);
border-radius: 15px;                     /* &lt;----  20px  */ 
border: 2px outset #094065;</string>
    </property>
    <property name="text">
     <string>Simulate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>730</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(132, 132, 132);
border-radius: 15px;                     /* &lt;----  20px  */ 
border: 2px outset #094065;</string>
    </property>
    <property name="text">
     <string>Claer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>730</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(132, 132, 132);
border-radius: 15px;                     /* &lt;----  20px  */ 
border: 2px outset #094065;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lines_3">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>490</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>segments</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>360</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> PROFILE VISIBILITY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="geo_base_en_2">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>410</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>geo base</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="node_points_en_2">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>450</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>node points</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="text_4">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>570</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>text</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="text_2">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>530</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>way point</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="text_3">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>530</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>way point</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>treeView</zorder>
   <zorder>planeView</zorder>
   <zorder>profileView</zorder>
   <zorder>label</zorder>
   <zorder>geo_base_en</zorder>
   <zorder>node_points_en</zorder>
   <zorder>text</zorder>
   <zorder>lines</zorder>
   <zorder>nodeDetails</zorder>
   <zorder>deleteButton</zorder>
   <zorder>addTurnNodeButton</zorder>
   <zorder>addNodeToSegmentButton</zorder>
   <zorder>selectItemButton</zorder>
   <zorder>moveNodeButton</zorder>
   <zorder>saveButton</zorder>
   <zorder>simulateButton</zorder>
   <zorder>clearButton</zorder>
   <zorder>backButton</zorder>
   <zorder>lines_3</zorder>
   <zorder>label_2</zorder>
   <zorder>geo_base_en_2</zorder>
   <zorder>node_points_en_2</zorder>
   <zorder>text_4</zorder>
   <zorder>text_2</zorder>
   <zorder>text_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1536</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
